fix: load cms page association data with category has no cms page (#15215)
shopware/platform: UPGRADE-6.8.md

<details>

## Default CMS page ID now persisted for categories

The default CMS page ID is now automatically written to the database when a category is saved without a `cmsPageId`.

> The runtime-only field `cmsPageIdSwitched` on `CategoryDefinition` will be removed without replacement.

Reviewer: ```suggestion
The runtime-only field `cmsPageIdSwitched` on `CategoryDefinition` was removed without replacement.
```

> ## Default CMS page ID now persisted for categories
> 
> Now the default CMS page ID is automatically written to the database when a category is saved without a `cmsPageId`. A migration also backfills all existing categories that have no CMS page assigned.
> 
> The runtime-only field `cmsPageIdSwitched` on `CategoryDefinition` will be removed without replacement.
> 

Reviewer: ```suggestion
## Default CMS page ID now persisted for categories

The default CMS page ID is now automatically written to the database when a category is saved without a `cmsPageId`.

The runtime-only field `cmsPageIdSwitched` on `CategoryDefinition` was removed without replacement.

```

in this file you need to write like the change already happend. this is what people read after 6.8 was released
